proto/agent/mocks/agent_mock.go

```diff
@@ -23,7 +23,7 @@ import (
 	context "context"
 	reflect "reflect"
 
-	gomock "github.com/golang/mock/gomock"
+	gomock "go.uber.org/mock/gomock"
```

Comment:
Did you run `make gen` or `make mock_gen` to generate this? What version of mockgen did you install?

I haven't tried it, but if appropriate update the mockgen reference (https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/README.md#mockgen) to https://github.com/uber-go/mock